docs/src/components/layouts/utility-page-layout/utility-info-table.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as utilityModules from '../../../data/utility-modules.json';
+import { withPrefix } from 'gatsby';
+
+const PATH_TO_UTILITY_PAGE = withPrefix('/foundations/example-utility');
+
+const renderStateModifiers = (activeStates) => {
+	// TODO: Make these links to the how to use utility page.
+	if (activeStates.length === 0) {
+		return <>None</>;
+	}
+	const states = [];
+	if (activeStates.includes('active')) {
+		states.push(<a href={PATH_TO_UTILITY_PAGE}>Active</a>);
+	}
+	if (activeStates.includes('hover')) {
+		states.push(<a href={PATH_TO_UTILITY_PAGE}>Hover</a>);
+	}
+	if (activeStates.includes('focus')) {
+		states.push(<a href={PATH_TO_UTILITY_PAGE}>Focus</a>);
+	}
+	if (activeStates.includes('visited')) {
+		states.push(<a href={PATH_TO_UTILITY_PAGE}>Visited</a>);
+	}
+	return states;
+};
+
+/**
+ * Draws the Utility Information Table holding the state modifiers and sizes.
+ * @param {string} param0.utilityModuleName The name of the utility module to draw.
+ * @returns
+ */
+const UtilityInfoTable = ({ utilityModuleName }) => {
+	const utilityInfo = utilityModules[utilityModuleName];
+	// This formula was borrowed from USWDS documentation site, and translated into JS
+	const gzipSize =
+		Math.round((utilityInfo.size * 0.185 + Number.EPSILON) * 100) / 100;
+
+	if (!utilityInfo) {
+		return (
+			<div>Error, {utilityModuleName}, utility information not found.</div>
+		);
+	}
+	return (
+		<table className="usa-table" width="100%">
+			<caption>Utility classes</caption>
```

Comment:
```suggestion
			<caption class="usa-sr-only">Utility module usage information</caption>
```

Let's hide this for sighted users and give it a better description for screen reader users.